content/api/commands/session.md

```diff
@@ -14,14 +14,15 @@ in order to reduce test setup times.
 Experimental</strong>
 
 The `session` API is currently experimental, and can be enabled by setting the
-[`experimentalSessionSupport`](/guides/references/experiments) flag to `true` in
-the Cypress config or by using [`Cypress.config()`](/api/cypress-api/config) at
-the top of a spec file.
+[`experimentalSessionAndOrigin`](/guides/references/experiments) flag to `true`
+in the Cypress config. Please note, if you already have the
```

Comment:
Usually we don’t put notes in the docs about deprecated or removed things unless it’s in the History section at the bottom of the doc.